lib/tests/formulae.rb

```diff
@@ -109,7 +109,7 @@ def detect_formulae!(args:)
 
         if diff_start_sha1.present? && diff_end_sha1.present?
           merge_base_sha1 =
-            Utils.safe_popen_read(git, "-C", repository, "merge-base",
+            Utils.popen_read(git, "-C", repository, "merge-base",
```

Comment:
If this works can you leave a comment above with e.g. `# Allow this to fail on e.g. shallow clones.`